demos/keyboard_nav/index.html

```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Blockly Demo: Keyboard Navigation</title>
+  <script src="../../blockly_compressed.js"></script>
+  <script src="../../blocks_compressed.js"></script>
+  <script src="../../javascript_compressed.js"></script>
+  <script src="../../msg/js/en.js"></script>
+  <style>
+    body {
+      background-color: #fff;
+      font-family: sans-serif;
+    }
+    h1 {
+      font-weight: normal;
+      font-size: 140%;
+    }
+  </style>
+</head>
+<body>
+  <h1><a href="https://developers.google.com/blockly/">Blockly</a> &gt;
+    <a href="../index.html">Demos</a> &gt; Keyboard Navigation</h1>
+
+  <p>You can enter keyboard navigation mode by <b>shift clicking anywhere on the
```

Comment:
How do you feel about turning on keyboard nav when the page loads or having a button that turns it on and puts the cursor on the first block?

I'd still add shift + click to the list of controls as a way of forcing the cursor to a specific location.

Reply:
I like the idea of having a button because that is how I see people possibly incorporating it into their applications in the future.